tensorboard/plugins/projector/vz_projector/vz-projector-projections-panel.ts

```diff
@@ -205,7 +205,7 @@ class ProjectionsPanel extends LegacyElementMixin(PolymerElement) {
       if (this.dataSet && this.projector) {
         this.dataSet.perturbTsne();
         this.projector.notifyProjectionPositionsUpdated();
-        this.perturbInterval = setInterval(() => {
+        this.perturbInterval = window.setInterval(() => {
```

Comment:
I'm curious about this changing when it worked before.  Was there some new TS error?

Reply:
It was surprising to me too.

If I were to guess, since TF.js works in node, TSC is confused as to which version of `setInterval` this is referring to (node version does not return number but an object or something else; I forget). As such, we need to be a bit more explicit about which version of setInterval we are using by referring to `window` (as opposed to `global`) in our ts_library. 

Thoughts? Is this change still good?

Reply:
The change still lgtm to land.

I see others online that suggest simply using `window.setInterval` too [1].
[1] https://github.com/Microsoft/TypeScript/issues/30128